src/plugins/discover/public/application/components/doc_viewer/doc_viewer.scss

```diff
@@ -19,6 +19,18 @@
     padding-top: $euiSizeS;
   }
 
+  .kbnDocViewer__multifield_row {
```

Comment:
The nesting here can be removed since there is just the single declaration on the td. In other words, the selector can be changed to:

```
.kbnDocViewer__multifield_row:hover td {...}
```